requirements/dev.txt

```diff
@@ -1,3 +1,4 @@
 -r base.txt
 flake8
 pre-commit
+colorlog
```

Comment:
This one is already in base.txt:

https://github.com/learningequality/kolibri/blob/master/requirements/base.txt